quickannotator/api/v1/tile/__init__.py

```diff
@@ -48,6 +48,13 @@ class SearchTileArgsSchema(Schema):
     x2 = fields.Float(required=True)
     y2 = fields.Float(required=True)
 
+class SearchTileByPolygonArgsSchema(Schema):
+    image_id = fields.Int(required=True)
+    annotation_class_id = fields.Int(required=True)
+    hasgt = fields.Bool(required=False)
+    include_placeholder_tiles = fields.Bool(required=False, default=False)    # A placeholder tile is a placeholder tile that has not yet been created in the database.
```

Comment:
i thought we opted to eliminate the idea of placeholder tiles? if the tile needs to be returned, then we might as well create it and make it real

Reply:
Yes, I think we discussed in another thread just having this endpoint return tileids and letting the backend handle tile creation logic when the client calls an /infer endpoint on the tileid? Sound good?

Reply:
yup, that sounds familiar!